Maestro/test/Microsoft.DotNet.Maestro.SubscriptionValidation/packages.config

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="xunit" version="2.1.0" targetFramework="net452" />
+  <package id="xunit.abstractions" version="2.0.0" targetFramework="net452" />
+  <package id="xunit.assert" version="2.1.0" targetFramework="net452" />
+  <package id="xunit.core" version="2.1.0" targetFramework="net452" />
+  <package id="xunit.extensibility.core" version="2.1.0" targetFramework="net452" />
+  <package id="xunit.extensibility.execution" version="2.1.0" targetFramework="net452" />
+  <package id="xunit.runner.msbuild" version="2.1.0" targetFramework="net461" />
```

Comment:
I think it would be valuable for these tests to be run with the build, as is done for other repos like corefx, coreclr, and cli.  Using `xunit.runner.visualstudio` seems to be the simplest way to do that; with it, the following line in the csproj runs the tests:
`<xunit Assemblies="$(OutputPath)\$(AssemblyName).dll" />`

It's also possible to use `xunit.runner.console` in the build by using an Exec task, as described at the bottom of http://xunit.github.io/docs/running-tests-in-msbuild.html, but I don't think that at this time we need the extra flexibility provided by that approach.